src/Xamarin.Android.Tools.AndroidSdk/Properties/Resources.resx

```diff
@@ -0,0 +1,206 @@
+    <comment>App manifest does not have 'manifest' root element</comment>
+  </data>
+  <data name="XamarinAndroidTools_XAT0002" xml:space="preserve">
+    <value>Must specify a package name or assembly name</value>
+    <comment>Must specify a package name or assembly name</comment>
+  </data>
+  
+  <data name="XamarinAndroidTools_XAT0003" xml:space="preserve">
+    <value>Could not determine Android SDK location. Please provide {0}.</value>
+    <comment>Could not determine Android SDK location. Please provide {0}.</comment>
+  </data>
+  
+  <data name="XamarinAndroidTools_XAT0004" xml:space="preserve">
+    <value>Could not determine Java SDK location. Please provide {0}.</value>
+    <comment>Could not determine Java SDK location. Please provide {0}.</comment>
+  </data>
+  
+  <data name="XamarinAndroidTools_XAT0005" xml:space="preserve">
+    <value>Windows is not supported at this time.</value>
+    <comment>Windows is not supported at this time.</comment>
+  </data>
+  
+  <data name="XamarinAndroidTools_XAT0006" xml:space="preserve">
+    <value>No Microsoft OpenJDK could be found.  Please re-run the Visual Studio installer or manually specify the JDK path in settings.</value>
+    <comment>No Microsoft OpenJDK could be found.  Please re-run the Visual Studio installer or manually specify the JDK path in settings.</comment>
+  </data>
+
+  <data name="XamarinAndroidTools_XAT0007" xml:space="preserve">
+    <value>`{0}` must be a directory!</value>
+    <comment>`{0}` must be a directory!</comment>
+  </data>
+
+  <data name="XamarinAndroidTools_XAT0008" xml:space="preserve">
+    <value>Not a directory</value>
+    <comment>Not a directory</comment>
+  </data>
+  
+  <data name="XamarinAndroidTools_XAT0009" xml:space="preserve">
+    <value>Requested to get one string value when property `{0}` contains `{1}` values.</value>
+    <comment>Requested to get one string value when property `{0}` contains `{1}` values.</comment>
+  </data>
+    
+  <data name="XamarinAndroidTools_XAT0010" xml:space="preserve">
+    <value>Could not find required file `{0}` within `{1}`; is this a valid JDK?</value>
+    <comment>Could not find required file `{0}` within `{1}`; is this a valid JDK?.</comment>
+  </data>
+  
+  <data name="XamarinAndroidTools_XAT0011" xml:space="preserve">
+    <value>Unknown property key for value {0}!</value>
+    <comment>Unknown property key for value {0}!</comment>
+  </data>
+  
+  <data name="XamarinAndroidTools_XAT0012" xml:space="preserve">
+    <value>GetMacOSMicrosoftJdkPaths jdks override! {0}</value>
+    <comment>GetMacOSMicrosoftJdkPaths jdks override! {0}</comment>
+  </data>
+  
+  <data name="XamarinAndroidTools_XAT0013" xml:space="preserve">
+    <value>The directory `{0}`, via locator `{1}`, is not a valid JDK directory: {2}</value>
+    <comment>The directory `{0}`, via locator `{1}`, is not a valid JDK directory: {2}</comment>
+  </data>
```

Comment:
The `comment` field won't need to contain a copy of the `value`. It can instead be used to provide additional context about the string as needed.

While we're on the topic of this message, I'll suggest adjusting the wording to make it more actionable and to align it more closely with some other messages from xamarin-android and androidtools:

```xml
<data name="XamarinAndroidTools_XAT0013" xml:space="preserve">
  <value>An exception occurred while validating the Java SDK installation in '{0}' that was found while searching the paths in '{1}'. Ensure that the Android section of the Visual Studio options has a valid Java SDK directory configured. To use a custom SDK path for a command line build, set the 'JavaSdkDirectory' MSBuild property to the custom path. Exception: {2}</value>
  <comment>{0} - The path of the invalid installation
{1} - The search path
{2} - The exception message of the associated exception</comment>
</data>
```